### PR TITLE
Update alert-filters.json

### PR DESCRIPTION
**Pull request by sivathota226**

### CODE REVIEW

File: alert-filters.json [hunk 1]
```diff
@@ -1,32 +1,46 @@
 {
-  "add": [
+  “add”: [
```
Comment (girishkamat) [inline alert-filters.json]: I think the double quote character is wrong

Comment (sivathota226) [reply alert-filters.json]: corrected it now.

File: alert-filters.json [hunk 2]
```diff
@@ -2,31 +2,45 @@
...
+      "contextId": 1,
+      "url": "https: \/\/optimizationguide-pa\\.googleapis\\.com.*",
       "ruleId": 10020,
       "newLevel": -1,
       "urlIsRegex": "true"
     },
     {
       "contextId": 1,
-      "url": "https:\/\/optimizationguide-pa\\.googleapis\\.com.*",
+      "url": "https: \/\/optimizationguide-pa\\.googleapis\\.com.*",
```
Comment (girishkamat) [inline alert-filters.json]: Why do we have a space here?

File: alert-filters.json [hunk 3]
```diff
@@ -2,31 +2,45 @@
...
+    {
+      "contextId": 1,
+      "url": "https: \/\/.*\\.google-analytics\\.com.*",
+      "ruleId": 3,
+      "newLevel": -1,
+      "urlIsRegex": "true"
+    },
+    {
+      "contextId": 1,
+      "url": "https: \/\/optimizationguide-pa\\.googleapis\\.com.*",
```
Comment (girishkamat) [inline alert-filters.json]: Space here too, this will fail the regex

File: alert-filters.json [hunk 4]
```diff
@@ -2,31 +2,45 @@
...
       "contextId": 1,
-      "url": "https:\/\/optimizationguide-pa\\.googleapis\\.com.*",
+      "url": "https: \/\/.*\\.google-analytics\\.com\/.*",
+      "ruleId": 2,
+      "newLevel": -1,
+      "urlIsRegex": "true"
+    },
+    {
+      "contextId": 1,
+      "url": "https: \/\/.*\\.google-analytics\\.com.*",
```
Comment (girishkamat) [inline alert-filters.json]: This line and above too has a space, which needs removal

**Review by girishkamat — APPROVED** [review]

**Pull request merged**